quantecon/markov/approximation.py

```diff
@@ -332,7 +334,7 @@ def discrete_var(A,
     if rv is None:
         u = random_state.standard_normal(size=(sim_length-1, r))
     else:
-        u = rv.rvs(size=sim_length-1, random_state=random_state)
+        u = rv.rvs(size=(sim_length-1, r), random_state=random_state)
```

Comment:
My assumption was that `rv` is a "frozen" object (if it comes from `scipy.stats`).

```py
r = 3
rv = multivariate_normal(cov=np.identity(r))
rv.rvs(size=5)
```

```
array([[ 0.6345146 ,  1.24005431,  0.49748714],
       [ 0.70736992, -0.98265265,  1.62881673],
       [-1.46650587, -1.30473345, -0.16520163],
       [-1.51102623, -0.10933109, -0.94159965],
       [ 0.29775017, -0.35714403,  0.28893599]])
```

```py
type(rv)
```

```
scipy.stats._multivariate.multivariate_normal_frozen
```

I wasn't aware that the classes had an `rvs` method. But for some classes, the mean and cov are hard to control?

Maybe we should add an example.

Reply:
Many thanks @oyamad for letting me know.

I have adjusted the code and the test to fit into the convention.

I directly used your example to show how to use the `rv` parameter as I think it would be slightly redundant if we have another example.

Please kindly let me know what you think about the current changes, and if it is needed to have another example.

Reply:
Thanks, but I guess the case with `rv` shouldn't be the main example. (Passing `scipy.stats.multivariate_normal` is redundant.) What would be a "primary" example for the noise term other than normal?

Reply:
I don't think there's an obvious primary example aside from normal.  But how https://en.wikipedia.org/wiki/Multivariate_t-distribution for an example?

Reply:
Many thanks @oyamad and @jstac for your suggestions. 

I just updated the docstring to include an example using multivariate t-distribution. 

Please kindly let me know if it needs any changes.